### ISSUE TITLE
Device battery?

### ISSUE THREAD

**Issue by robkirk** [cyberjunky/home-assistant-garmin_connect#48]
Are there any metrics that can be accessed regarding the device itself?

In particular, is it possible to get the device battery percentage? It would be nice to have an automation setup to remind me to charge it when it gets below a certain threshold.

Thanks

**Comment by teranex** [comment]
This is exactly what I'm looking for as well as I always forget to charge my Venu2+

**Comment by cyberjunky** [comment]
I don't think so, since it's not synced to Garmin Connect website, closing until new information or feature are added to the Garmin Connect website.
